src/applications/edu-benefits/0993/config/form.js

```diff
@@ -95,7 +95,7 @@ const formConfig = {
               }
             },
             'view:optOutMessage': {
-              'ui:title': tabIndexedTitle('By clicking the Continue button, you’re asking VA to not share your education benefits information.')
+              'ui:field': () => tabIndexedTitle('By clicking the Continue button, you’re asking VA to not share your education benefits information.')
```

Comment:
`ui:title` includes a `<legend>` which was interfering with one of the accessibility tests.

Reply:
Good to know, we will probably need to apply this pattern/fix, in many other places.